anomaly.json

```diff
@@ -0,0 +1,53 @@
+{
+    "id": "anomaly",
```

Comment:
seems like very generic name for a specific use case. Shouldn't this be namespaced a bit, like e.g. " climatological_normal_anomaly"?

Reply:
We talked to TU Vienna and they said the process is broad enough to also support EO use cases so it's not only meant for a specific use case.